dictionary.js

```diff
@@ -0,0 +1,44 @@
+const dictionary = [];
+
+function addWord(event) {
+    event.preventDefault();
+    const word = add.value;
+    if (!dictionary.includes(word)) {
+        dictionary.push(word);
+        const listElem = list;
+        const newWord = document.createElement('li');
+        newWord.textContent = word;
+        listElem.appendChild(newWord);
+        newWord.style.display = 'none';
+        alert("You have successfully added '" + word + "' to the dictionary.");
+    } else {
+        alert("The word '" + word + "' is already in the dictionary.");
+    }
+    add.value = "";
+}
+
+function searchWord(event) {
+    event.preventDefault();
+    let searchedWord = search.value;
+    const elements = list;
+    const childElements = elements.children;
+    for (let i = 0; i < dictionary.length; ++i) {
```

Comment:
You can use 'includes' and 'filter' predefined functions in order to have a better logic of the code